### ISSUE TITLE
Implement archetype effect: Draconic Knight - Wall climbing

### ISSUE THREAD

**Issue by Christian-Toney** [EveryoneDestroysTheWorld/game#108]
# Dependencies
- [x] #189 

# Related design entries
* https://github.com/EveryoneDestroysTheWorld/design/blob/main/inventory/archetypes/draconic-knight.md

# Completion criteria
- [ ] Let the player hang on walls.
- [ ] Reasonably reduce stamina while the player hangs on a wall.
- [ ] Let the player climb walls.
- [ ] Reasonably reduce stamina while the player climbs a wall.
- [ ] Launch off the wall while the player is on a wall.

**Comment by zriku1** [comment]
Is this nessesary with flight? maybe it uses less stamina for the same distance?

**Comment by Christian-Toney** [comment]
You're totally right. It's not necessary for the prototype, but it was a detail I had in mind since if they could fly, why not grapple to walls too? 

I'll move this to v1 so we can decide if we want to keep it.

**Comment by zriku1** [comment]
Maybe could add it to dive bomb, where selecting a wall lets you grapple to it and wall grab, then you can do a followup dive bomb for no extra stam cost. I like the idea of dk being slow but having most of their stam going to strong mobility options. Like they mostly move around slowly but when they need to get somewhere fast they can spend a bunch of stam to close the distance
